tdrs-backend/tdpservice/parsers/schema_defs/tanf/t1.py

```diff
@@ -11,98 +10,266 @@
         Field(item="26AIV", name='SANC_TEEN_PARENT', type='number', startIndex=98, endIndex=99,
-              required=True, validators=[]),
+              required=True, validators=[
+                  validators.oneOf([1, 2]),
+              ]),
         Field(item="26AV", name='NON_COOPERATION_CSE', type='number', startIndex=99, endIndex=100,
-              required=True, validators=[]),
+              required=True, validators=[
+                  validators.oneOf([1, 2]),
+              ]),
         Field(item="26AVI", name='FAILURE_TO_COMPLY', type='number', startIndex=100, endIndex=101,
-              required=True, validators=[]),
+              required=True, validators=[
+                  validators.oneOf([1, 2]),
+              ]),
         Field(item="26AVII", name='OTHER_SANCTION', type='number', startIndex=101, endIndex=102,
-              required=True, validators=[]),
+              required=True, validators=[
+                  validators.oneOf([1, 2]),
+              ]),
         Field(item="26B", name='RECOUPMENT_PRIOR_OVRPMT', type='number', startIndex=102, endIndex=106,
-              required=True, validators=[]),
+              required=True, validators=[
+                  validators.isLargerThanOrEqualTo(0),
+              ]),
         Field(item="26CI", name='OTHER_TOTAL_REDUCTIONS', type='number', startIndex=106, endIndex=110,
-              required=True, validators=[]),
+              required=True, validators=[
+                  validators.isLargerThanOrEqualTo(0),
+              ]),
         Field(item="26CII", name='FAMILY_CAP', type='number', startIndex=110, endIndex=111,
-              required=True, validators=[]),
+              required=True, validators=[
+                  validators.oneOf([1, 2]),
+              ]),
         Field(item="26CIII", name='REDUCTIONS_ON_RECEIPTS', type='number', startIndex=111, endIndex=112,
-              required=True, validators=[]),
+              required=True, validators=[
+                  validators.oneOf([1, 2]),
+              ]),
         Field(item="26CIV", name='OTHER_NON_SANCTION', type='number', startIndex=112, endIndex=113,
-              required=True, validators=[]),
-        Field(item="27", name='WAIVER_EVAL_CONTROL_GRPS', type='number', startIndex=113, endIndex=114,
-              required=True, validators=[]),
+              required=True, validators=[
+                  validators.oneOf([1, 2]),
+              ]),
+        Field(item="27", name='WAIVER_EVAL_CONTROL_GRPS', type='string', startIndex=113, endIndex=114,
+              required=True, validators=[
+                  validators.or_validators(validators.matches('9'), validators.isEmpty()),
+                  validators.isAlphaNumeric(),
+              ]),
         Field(item="28", name='FAMILY_EXEMPT_TIME_LIMITS', type='number', startIndex=114, endIndex=116,
-              required=True, validators=[]),
+              required=True, validators=[
+                  validators.oneOf([1, 2, 3, 4,
```

Comment:
does it matter that this data element is a string type? is `01` the same as `1` here, for example?

Reply:
by changing type to number, we are getting rid of leading zeros, so 01 is equal to 1